tensorflow_lite_support/web/tflite_model_runner/cc/tflite_model_runner.h

```diff
@@ -37,9 +37,11 @@ struct TFLiteWebModelRunnerOptions {
 
   bool enable_webnn_delegate = false;
 
-  // 0 - default, 1 - gpu, 2 - cpu
+  // 0 - cpu, 1 - gpu, 2 - npu
```

Comment:
"npu" is not supported by Chromium implementation yet, setting it would get exception.

Reply:
Yeah, I know that, it's a placeholder for further NPU support. (Specific for ChromeOS team for testing) It won't throw if we don't set it to 'npu'. This can be controlled at the user end. (e.g. We can just expose `cpu` and `gpu` options to e2e benchmark)